src/test/java/swift/storage/JsonAdaptedBridgeTest.java

```diff
@@ -0,0 +1,60 @@
+package swift.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static swift.storage.JsonAdaptedBridge.MISSING_FIELD_MESSAGE_FORMAT;
+import static swift.testutil.Assert.assertThrows;
+import static swift.testutil.TypicalBridges.DEFAULT_BRIDGE_1;
+
+import java.util.UUID;
+
+import org.junit.jupiter.api.Test;
+
+import swift.commons.exceptions.IllegalValueException;
+
+public class JsonAdaptedBridgeTest {
+    private static final String INVALID_PERSON_ID = "invalid";
+    private static final String INVALID_TASK_ID = "invalid";
+
+    private static final String valid_person = DEFAULT_BRIDGE_1.getPersonId().toString();
```

Comment:
Variable should be capitalized as it is a constant i.e. VALID_PERSON :)

Reply:
thanks! good catch